examples/api/main.py

```
@@ -71,7 +69,46 @@ class Meta:
+    return bcrypt.hashpw(password.encode(), bcrypt.gensalt()).decode()
+
+
+def check_password(hashed_password: str, password: str) -> bool:
+    return bcrypt.checkpw(password.encode(), hashed_password.encode())
+
+
+def jwt_middleware(request, next, **kwargs):
+    token = request.headers.get("authorization", "").replace("Bearer ", "")
+
+    if token:
+        if claims := jwt.verify_token(token):
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)